modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/odata/filter/expression/PredicateExpressionVisitorImpl.java

```diff
@@ -740,6 +750,38 @@ private Object _getValue(
 		}
 	}
 
+	private Object _handleComplexPropertyExpressionPropertyExpression(
```

Comment:
This name seems a bit long :thinking: 

What about `_handleComplexPropertyExpression`

Reply:
Well, I'm just following the pattern of naming things from that come from (we use that in Liferay 🤷 )
Also, the method does not handle a ComplexPropertyExpression, it handles the property expression stored in a ComplexPropertyExpression. Maybe `_handlePropertyExpression` makes sense, but it is only for the property expressions from ComplexPropertyExpression 🤔 🤔 🤔 

I will try to reduce the name and see what Brian says.

Reply:
Done